theano/gof/tests/test_graph_opt_caching.py

```diff
@@ -6,10 +6,8 @@
 import theano
 import theano.tensor as T
 
-
-
 def test_graph_opt_caching():
-    opt_db_file = theano.config.compiledir+'/optimized_graphs.pkl'
+    opt_db_file = theano.config.compiledir+'/*.pkl'
     os.system('rm %s'%opt_db_file)
```

Comment:
You should write setup() and teardown() methods that change the default compiledir (perhaps to a subdir of the current one) and make sure to remove it after.  This way you can isolate the test from the rest of the run.